ls1intum/Apollon
# `Development`: Fix issue for exporting manually layouted diagram

### Checklist


### Motivation and Context
This PR fixes an issue with the cut-off of elements of a manually layouted diagram.

### Description
Issue Number: https://github.com/ls1intum/Apollon/issues/258

### Steps for Testing
1. Clone the repo
2. Checkout to branch `bugfix/export-fix`
3. Start the application by executing `yarn install` followed by `yarn start`
4. Insert any two elements and connect them in the canvas in a way that they have manual waypoint connectors (more than 2 edges).
5. Layout the connector manually
6. Export the diagram by clicking on the All button from Export Menu on side menu
7. Observe that all diagram elements are visible with no cut-off.

### Screenshots
#### Before:
![screen-capture (15)](https://user-images.githubusercontent.com/14681902/186425350-01b65e17-a58e-424f-8584-d1070900bbb5.gif)

#### After:
![screen-capture (14)](https://user-images.githubusercontent.com/14681902/186424749-c30391e1-397e-43e0-a332-b2a3093abf32.gif)
